app/main/views/buyers.py

```diff
@@ -54,12 +56,17 @@ def update_brief(brief_id):
                                   'applications_closed_at': request.form['closed_at']
                                   },
                        'update_details': {'updated_by': current_user.email_address}}).get('briefs')
-        elif request.form.get('edit_seller_email_list'):
-            edit_seller_email_list = request.form['edit_seller_email_list'].split(get_new_line())
+        elif 'edit_seller_email_list' in request.form:
+            edit_seller_email_list = request.form.get('edit_seller_email_list', []).split(get_new_line())
             brief = data_api_client.req.briefs(brief_id).admin() \
                 .post({'briefs': {'sellerEmailList': [_.strip() for _ in edit_seller_email_list if _ != '']
                                   },
                       'update_details': {'updated_by': current_user.email_address}}).get('briefs')
+        elif 'edit_area_of_expertise' in request.form:
+            brief = data_api_client.req.briefs(brief_id).admin() \
+                .post({'briefs': {'areaOfExpertise': request.form.get('edit_area_of_expertise', None)
```

Comment:
nitpick - you don't need to do a `.get('edit_area_of_expertise', None)` if you've already checked `'edit_area_of_expertise' in request.form` you can just do `request.form['edit_area_of_expertise']`

Reply:
updated to use key directly.